### PR TITLE
Release/3.14.2

### PR DESCRIPTION
**Pull request by joshdholtz**
- Deprecated `Purchases` pod in favor of `RevenueCat`
    https://github.com/RevenueCat/purchases-ios/pull/1552

### CODE REVIEW

**Review by NachoSoto — APPROVED** [review]

**Comment by joshdholtz** [comment]
@NachoSoto The Carthage integration tests are failing because of reasons that I don't _think_ are super important but 🤷‍♂️  Is this still okay to merge and release? 🙃

**Comment by NachoSoto** [comment]
What's the failure?

**Comment by joshdholtz** [comment]
@NachoSoto It looks like its checking out `4.3.0` so Magic Weather is failing because its built with v3 stuff on this branch 😛

**Comment by NachoSoto** [comment]
Looking at that now it seems wrong. [The new version](https://github.com/RevenueCat/purchases-ios/blob/main/Tests/InstallationTests/CarthageInstallation/fastlane/Fastfile) correctly replaces `main` with the right commit.
But [in 3.x this seems wrong](https://github.com/RevenueCat/purchases-ios/blob/3.14.1/IntegrationTests/CarthageIntegration/fastlane/Fastfile), it's replacing `CARTHAGE_INTEGRATION_TESTS_GIT_COMMIT` but [it's not there](https://github.com/RevenueCat/purchases-ios/blob/3.14.1/IntegrationTests/CarthageIntegration/Cartfile)?
I would recommend fixing, maybe Carthage tests haven't worked for a while in 3.x releases?

**Comment by joshdholtz** [comment]
@NachoSoto Looks like we gucci now!

File: IntegrationTests/CarthageIntegration/Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-git "./purchases-root"
\ No newline at end of file
+git "./purchases-root" "CARTHAGE_INTEGRATION_TESTS_GIT_COMMIT"
```
Comment (NachoSoto) [inline IntegrationTests/CarthageIntegration/Cartfile:1]: ❤️

**Review by NachoSoto — APPROVED** [review]
Yay!

**Pull request merged**